gormigrate.go

```diff
@@ -177,23 +214,37 @@ func (g *Gormigrate) getLastRunnedMigration() (*Migration, error) {
 
 // RollbackMigration undo a migration.
 func (g *Gormigrate) RollbackMigration(m *Migration) error {
+	return g.rollbackMigration(m, true)
+}
+
+// rollbackMigration rolls back the given transaction. If `withTransaction` is true, then the operation is wrapped in
+// transaction (if g.options.UseTransaction is also true).
+// Call rollbackMigration with `withTransaction` false if you want to wrap multiple operations in a single transaction.
+func (g *Gormigrate) rollbackMigration(m *Migration, withTransaction bool) error {
```

Comment:
This `withTransaction` parameter is useless, since only called with `true`, and can be removed

Reply:
Not really: take a look at the line 195.

Reply:
It was done to avoid starting transaction inside an existing transaction.
I removed the flag and slightly changed the code.